README.md

```diff
@@ -2,3 +2,19 @@
 
 [![Build Status](https://travis-ci.org/hmcts/bulk-scan-orchestrator.svg?branch=master)](https://travis-ci.org/hmcts/bulk-scan-orchestrator)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e9272daf4b714e4f95280916e763b6b2)](https://www.codacy.com/app/HMCTS/bulk-scan-orchestrator)
+
+## Running end to end using docker
+- Run up the docker environment from bulk-scan-shared-infrastructure
+- You will have to setup these environment variables on either your
+run configuration or bash shell
+```$bash
+IDAM_USERS_SSCS_USERNAME = bulkscanorchestrator+systemupdate@gmail.com
+IDAM_USERS_SSCS_PASSWORD = Password12
+CORE_CASE_DATA_API_URL = http://localhost:4452
+```
+- add a case into using the docker ui
```

Comment:
How about?
> add a case into ccd using the case management ui http://localhost:3451

Reply:
yeah ... was going to add that but could not remember the port at the time of writing.
will add it.